tests/flow/test_graph_merge.py

```
@@ -0,0 +1,699 @@
+    def test26_merge_set_invalid_property(self):
+        self.graph.delete()
+        query = """MATCH p=() MERGE () ON MATCH SET p.prop4 = 5"""
+        result = self.graph.query(query)
+        self.env.assertEquals(result.properties_set, 0)
+
+    def test27_merge_create_invalid_entity(self):
+        try:
+            # Try to create a node with an invalid NULL property.
+            query = """MERGE (n {v: NULL})"""
+            self.graph.query(query)
+            assert(False)
```

Ruff (0.12.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)